pruning/src/lib.rs

```diff
@@ -20,9 +20,10 @@
 
 use std::cmp::Ordering;
 
+use cuprate_constants::block::MAX_BLOCK_HEIGHT_USIZE;
+
 use thiserror::Error;
 
-pub const CRYPTONOTE_MAX_BLOCK_HEIGHT: usize = 500000000;
```

Comment:
`CRYPTONOTE_MAX_BLOCK_HEIGHT` is replaced by `cuprate_constants::block::MAX_BLOCK_HEIGHT[_USIZE]` throughout the codebase.

Also, I think Cuprate is far enough removed from the original CryptoNote that we can start dropping the `CRYPTONOTE_` prefix.